repos/spack_repo/builtin/packages/py_tables/package.py

```diff
@@ -17,7 +17,7 @@ class PyTables(PythonPackage):
 
     license("BSD-3-Clause")
 
-    version("master", branch="master")
+    version("master", branch="master", submodules=True)
```

Comment:
Why don't we add the latest [v3.10.2](https://github.com/PyTables/PyTables/releases/tag/v3.10.2) release too?